### PR TITLE
2860 add dark mode toggle into oxidation/client

### LINKED ISSUE #2860: (Ionic) Add a parameter to toggle Dark mode (dark, light and system)
https://ionicframework.com/docs/theming/dark-mode#manually-toggle-dark-mode

### PR DESCRIPTION
**Pull request by Landeers**
fix #2860 

Works on OS, browser and android.
Don't know why on android, dark-mode is enabled by default even if emulated system is not.

### CODE REVIEW

**Review by NicoTuxx — APPROVED** [review]

**Comment by NicoTuxx** [comment]
Mutualize the dark mode toggle function

File: oxidation/client/android/app/src/main/res/xml/config.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version='1.0' encoding='utf-8'?>
 <widget version="1.0.0" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
   <access origin="*" />
-
-
-</widget>
+  
+  
+</widget>
```
Comment (FirelightFlagboy) [inline oxidation/client/android/app/src/main/res/xml/config.xml:6]: missing newline

did you configure [editorconfig](https://editorconfig.org/) ?

**Review by FirelightFlagboy — CHANGES_REQUESTED** [review]

File: oxidation/client/android/app/src/main/res/xml/config.xml (hunk 1 above)
Comment (Landeers) [reply oxidation/client/android/app/src/main/res/xml/config.xml]: It's a generated file, we should use a precommit to fix it automaticaly

**Comment by Landeers** [comment]
> Mutualize the dark mode toggle function

done

File: oxidation/client/android/app/src/main/res/xml/config.xml (hunk 1 above)
Comment (FirelightFlagboy) [reply oxidation/client/android/app/src/main/res/xml/config.xml]: https://github.com/Scille/parsec-cloud/issues/2884

**Review by FirelightFlagboy — APPROVED** [review]

**Review by FirelightFlagboy — APPROVED** [review]

**Pull request merged**